### DISCUSSION TITLE
A service to reload integrations

### DISCUSSION THREAD

**Discussion by GentleHoneyLover** [frenck/spook#737] [Ideas] [answered]
Hey! Love your custom integration!

It already provides services to enable or disable integrations. It would be great to have a service that reloads an integration.

Use cases:
1. Sometimes an integration gets stuck and cannot connect to a device or a cloud service. A reload often fixes that.
2. If you add a new device, you need to reload the HomeKit bridge integration in order to expose that new device to HomeKit.

Thanks!

**Comment by frenck** [comment, marked as answer]
That already exists natively in Home Assistant.

`homeassistant.reload_config_entry`
